### PR TITLE
update packages

### PR DESCRIPTION
**Pull request by onnovisser**

### CODE REVIEW

File: src/components/crowdloan-user/User.tsx [hunk 1]
```diff
@@ -16,11 +16,15 @@ import { getAccountDetails, useDidClaim, useTotalRewards } from './utils'
...
   const centrifuge = useCentrifuge()
-  const { selectedAccount } = useWallet()
+  const {
+    substrate: { selectedAccount },
+  } = useWallet()
   const didClaim = useDidClaim(selectedAccount?.address)
   const totalRewards = useTotalRewards({ address: selectedAccount?.address, parachain: centrifuge.config.network })
   const currency = centrifuge.config.network === 'altair' ? 'AIR' : 'CFG'
 
+  console.log('didClaim', didClaim)
```
Comment (sophialittlejohn) [inline src/components/crowdloan-user/User.tsx:26]: remove?

File: src/components/crowdloan-user/index.tsx [hunk 2]
```diff
@@ -11,7 +11,7 @@ export default function CrowdloanUser({ network }: CrowdloanUserProps) {
   const centConfig: UserProvidedConfig = React.useMemo(
     () => ({
       network,
-      ...(process.env.NODE_ENV === 'development' && {
+      ...(process.env.NODE_ENV === 'developments' && {
```
Comment (sophialittlejohn) [inline src/components/crowdloan-user/index.tsx:14]: are you sure the s belongs there?

File: src/theme.ts [hunk 3]
```diff
@@ -25,20 +25,20 @@ export const theme: DefaultTheme = {
       letterSpacing: '-0.04em',
       color: 'textPrimary',
     },
-    heading2: {
+    heading2b: {
```
Comment (sophialittlejohn) [inline src/theme.ts:28]: why did you have to include the b's here?

File: src/components/crowdloan-user/index.tsx (hunk 2 above)
Comment (onnovisser) [reply src/components/crowdloan-user/index.tsx:14]: not sure how this one happened 😅

File: src/theme.ts (hunk 3 above)
Comment (onnovisser) [reply src/theme.ts:28]: the headings from the fabric theme that it was overriding were messing up Fabric components like the WalletDialog. So I just renamed the ones that were messing things up

**Pull request merged**